frontend/src/components/PaginatedTable.js

```diff
@@ -51,7 +51,7 @@ function PaginatedTable ({
 
   if (shouldLoadNextChunk) {
     const offset = Math.floor(startIndex / pageSize) * pageSize;
-    setTimeout(() => onLoad({ offset, filters, sortBy, filterKey }), 0);
```

Comment:
i'm not sure how to reproduce this on my master branch, but it looks like a reasonable fix.